modular_skyrat/modules/loadouts/loadout_items/loadout_datum_pocket.dm

```diff
@@ -172,7 +172,85 @@ GLOBAL_LIST_INIT(loadout_pocket_items, generate_loadout_items(/datum/loadout_ite
 /datum/loadout_item/pocket_items/cross
 	name = "Ornate Cross"
 	item_path = /obj/item/crucifix
-	restricted_roles = list(JOB_CHAPLAIN)
+
+/*
+*	UTILITY
+*/
+
+/datum/loadout_item/pocket_items/toolbox
+	name = "Full Toolbox"
+	item_path = /obj/item/storage/toolbox/mechanical
+
+/datum/loadout_item/pocket_items/toolbox/insert_path_into_outfit(datum/outfit/outfit, mob/living/carbon/human/equipper, visuals_only = FALSE, override_items = LOADOUT_OVERRIDE_BACKPACK)
```

Comment:
This shouldn't be in pockets, though, this should be a inhand thing, it's a big item and shouldn't normally be fitting in the pockets.

I also think that there should be a restriction from this being given to prisoners, I can't remember if we got code to blacklist an item from a job, if not, I think this should be a reason to add code to blacklist an item from a job.

Reply:
Real.